### PR TITLE
dynamic generate shlp dialog

### PR DESCRIPTION
**Pull request by choper725**
a first take on dynamic generate shlp dialog

### CODE REVIEW

File: src/z2ui5_cl_xml_view.clas.abap [hunk 1]
```diff
@@ -2822,6 +2839,285 @@ CLASS Z2UI5_CL_XML_VIEW IMPLEMENTATION.
...
+        lfieldname(132) TYPE c,
+        ltrflddis(1)    TYPE c,
+        bidictrlc(1)    TYPE c,
+        outputstyle(2)  TYPE n,
+        nohistory(1)    TYPE c,
+        ampmformat(1)   TYPE c,
+    END OF ty_ddfields.
+
+     DATA: lv_grid_form_no     TYPE i,
+           lt_arg              TYPE TABLE OF string,
```
Comment (larshp) [inline src/z2ui5_cl_xml_view.clas.abap:2909]:
```suggestion
           lt_arg              TYPE STANDARD TABLE OF string WITH EMPTY KEY,
```

**Comment by oblomov-dev** [comment]
thank you! added some abaplint fixes.

**Comment by oblomov-dev** [comment]
@choper725 updated the naming of custom controls and functions to separate them from the standard ui5 functionality and controls

**Comment by choper725** [comment]
i sent another PR fixing demo 102 and adding ircontroller to zcc_generate_ddic_shlp in order to get the sending class object..

ill try to think on a way integrating the select data inside this custom control also... and not seperating it to two classes..

**Pull request merged**